Orange/widgets/regression/owlinearregression.py

```diff
@@ -171,6 +171,27 @@ def commit(self):
         self.send("Coefficients", coef_table)
 
     def send_report(self):
+        self.report_items((("Name", self.learner_name),))
+
+        if self.reg_type == OWLinearRegression.OLS:
+            regularization = "No Regularization"
+        elif self.reg_type == OWLinearRegression.Ridge:
+            regularization = ("Ridge Regression (L2) with α={}"
+                             .format(self.alphas[self.alpha_index]))
+        elif self.reg_type == OWLinearRegression.Lasso:
+            regularization = ("Lasso Regression (L1) with α={}"
+                             .format(self.alphas[self.alpha_index]))
+        elif self.reg_type == OWLinearRegression.Elastic:
+            regularization = ("Elastic Net Regression with α={}"
+                             " and L1:L2 ration of {}:{}"
+                             .format(self.alphas[self.alpha_index],
+                                     self.l1_ratio,
+                                     1 - self.l1_ratio))
+
+        self.report_items((("Regularization", regularization),))
```

Comment:
Please put both, Name and Regularization into the same `report_items` to avoid the empty space between them.